packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -332,11 +332,14 @@ class DynamicSidecarServiceLabels(BaseModel):
         description="allow complete internet access to containers in here",
     )
 
-    callbacks_mapping: Json[CallbacksMapping] | None = Field(
-        default_factory=CallbacksMapping,  # type: ignore[arg-type] # this one ANE I am not sure about
-        alias="simcore.service.callbacks-mapping",
-        description="exposes callbacks from user services to the sidecar",
-    )
+    callbacks_mapping: Annotated[
+        Json[CallbacksMapping] | None,
+        Field(
+            default_factory=CallbacksMapping,  # this one ANE I am not sure about
```

Comment:
I think you can drop the comment. I think it's equivalent. Since the end object will be a CallbacksMapping object

Reply:
Left some extra comments. IMO we should review this setup at some point